tests/python/unit/test_model.py

```diff
@@ -121,3 +123,77 @@ def metrics_assertions(metrics, model_name):
     def forecasts_assertions(forecasts_df, model_name, prediction_length=1):
         assert len(forecasts_df.index) == 2
         assert forecasts_df["index"].nunique() == prediction_length
+
+
+class TestExternalFeaturesModel:
```

Comment:
Suggestion: Specify SimpleFeedForward in the test class name?

Reply:
done in 
d42615b